index.html

```diff
@@ -1011,6 +1011,33 @@ <h4>Exchange Examples</h4>
     <h2>Privacy Considerations</h2>
     <p>
     </p>
+
+    <section>
+      <h3>Phone Home Considered Harmful</h3>
+
+      <p>
+It is considered a bad practice for a <a>verifier</a> to contact an
```

Comment:
```suggestion
It is considered a privacy bad practice for a <a>verifier</a> to contact an
```

Reply:
If `privacy` is to be inserted, it should follow `bad`, not precede it.
```suggestion
It is considered a bad privacy practice for a <a>verifier</a> to contact an
```